packages/manager/apps/key-management-service/src/mocks/catalog/catalog.handler.ts

```diff
@@ -0,0 +1,23 @@
+import { Handler } from '../../../../../../../playwright-helpers';
```

Comment:
**question:** We will soon remove playwright from the repo.
Can you eliminate the dependency on it?

Reply:
This type is cross product so it should be set on MRC or another shared dependency to be available for all apps.

Reply:
For delete this dependencies we need to do have alternative of mock. I think can change the mock with Axios Mock : https://www.npmjs.com/package/axios-mock-adapter

And with this the test can be faster.

But to not create another new implementation. I agree with Romain. All units tests have this dependancies. 

It's possible to planned on control tower a example of implementation with the method of mock recommended ?

Maybe implementation on generator ?

Reply:
we will clean our dependencies to the playwright folder on a specific development.